src/js/profile/chart.js

```diff
@@ -237,7 +242,43 @@ export class Chart extends Observable {
       ],
     };
 
-    embed(this.container, barChart, { actions: false})
+
+    var handler = (h, event, item, value) => {
+      this.el = document.getElementById(DEFAULT_OPTIONS.id);
+      if (!this.el) {
+        this.el = document.createElement('div');
+        this.el.setAttribute('id', DEFAULT_OPTIONS.id);
+        this.el.classList.add('vg-tooltip');
```

Comment:
I think we already have a class for tooltips.

Reply:
Added the default class.